gmailservice/gmailservice_test.go

```diff
@@ -162,7 +162,7 @@ func TestDownloadFullMessages(t *testing.T) {
     verify429sAnd200s(expected.firstAdvance.failures, expected.firstAdvance.successes)
 
     // Advance fakeClock to first retry (still before the time we start responding w/ 200s)
-    fakeClock.Advance(30 * time.Second)
+    fakeClock.Advance(RetryWaitInterval * time.Second)
     // Wait until all three goroutines have called Sleep the second time
     fakeClock.BlockUntil(maxWorkers)
```

Comment:
It's stopping on this line. Possibly because we're not advancing the right number of seconds?